features/step_definitions/administration_steps.rb

```diff
@@ -0,0 +1,118 @@
+Given(/^I have an admin account$/) do
+  FactoryGirl.create(:admin_user)
+end
+
+When(/^I visit the admin panel url$/) do
+  visit new_admin_user_session_path
+end
+
+When(/^I enter my admin email$/) do
+  fill_in 'Email', with: 'admin@example.com'
+end
+
+When(/^I enter my admin password$/) do
+  fill_in 'Password', with: 'password'
+end
+
+Then(/^I see the admin panel$/) do
+  visit admin_root_path
+end
+
+Given(/^I do not have an admin account$/) do
+end
+
+Then(/^I see a flash notification that tell me that my email does not exist in the system$/) do
+  expect(page).to have_content("Invalid email or password.")
+end
+
+Given(/^I am logged into the admin panel$/) do
+  FactoryGirl.create(:admin_user)
+  visit new_admin_user_session_path
+  fill_in 'Email', with: 'admin@example.com'
+  fill_in 'Password', with: 'password'
+  click_on "Login"
+end
+
+Given(/^I am logged into the site$/) do
+  FactoryGirl.create(:user, email: "ruti@test.com")
+  ctoken = ActionMailer::Base.deliveries.last.body.match(/confirmation_token=\w*/)
```

Comment:
In general, it's great to write the cucumber steps from the perspective of the user, but in some case, it's very difficult and/or resource intensive to do that. This, I think, is one of those cases. Since you're going to be using this `Given` in a bunch of places, I think it's alright to take a shortcut and do `FactoryGirl.create(:user, email: 'ruti@test.com', confirmed_at: Date.current)` or something similar.